nemo_text_processing/g2p/data/heteronym_classification_data.py

```diff
@@ -65,98 +67,136 @@ def __init__(
         self.with_labels = with_labels
         self.wiki_homograph_dict = wiki_homograph_dict
         self.wordid_to_idx = wordid_to_idx
+        self.LOSS_PAD_TOKEN = -100
+        self.PAD_TOKEN = 0
 
```

Comment:
Should `self.labels_tkn2id` be initialized here as well? otherwise, there is no clue what is it as shown in,

```python
        for word_id, word in enumerate(text.split()):
            tokens.append(self.labels_tkn2id[word])
        return tokens
```